habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/exec/PythonVersionHelper.java

```diff
@@ -21,17 +22,17 @@ public class PythonVersionHelper {
 
     private static final String PYTHON_COMMAND = "python";
     private static final String PYTHON_3_COMMAND = "python3";
```

Comment:
I: some minor sonarlint tidying while I was here - this class is easier to review if you hide whitespace